controller/src/main/java/io/pravega/controller/metrics/StreamMetrics.java

```diff
@@ -96,6 +53,10 @@ public final class StreamMetrics extends AbstractControllerMetrics {
     private StreamMetrics() {
         createStreamLatency = STATS_LOGGER.createStats(CREATE_STREAM_LATENCY);
         deleteStreamLatency = STATS_LOGGER.createStats(DELETE_STREAM_LATENCY);
+        deleteStreamEventLatency = STATS_LOGGER.createStats(DELETE_STREAM_EVENT_LATENCY);
```

Comment:
Change the name of all these variables, as you may see `deleteStreamLatency` and `deleteStreamEventLatency` are too similar and confusing. According to my below comment on the metrics names, I would rename these metrics like:
`deleteStreamEventLatency -> controllerEventProcessorDeleteStreamLatency`.

Also, you may want to add them as a block at the end of these fields with some comment on top specifying what they are, like `// Write event latency of Controller EventProcessors` or something similar.

Reply:
updated names like "CONTROLLER_EVENT_PROCESSOR_DELETE_STREAM_LATENCY" , which will provide differentiation from existing operation related stats.